### PR TITLE
fixes for tests and cleanup

### PR DESCRIPTION
**Pull request by runcom**
@mrunalp PTAL some of our tests are failing with network timeouts (on slow networks) since we pull redis:alpine by digest in each test (slowing tests time down also)

### CODE REVIEW

File: test/helpers.bash [hunk 1]
```diff
@@ -168,6 +181,11 @@ function start_crio() {
 		"$BIN2IMG_BINARY" --root "$TESTDIR/crio" $STORAGE_OPTS --runroot "$TESTDIR/crio-run" --source-binary "$PAUSE_BINARY"
 	fi
 	"$COPYIMG_BINARY" --root "$TESTDIR/crio" $STORAGE_OPTS --runroot "$TESTDIR/crio-run" --image-name=redis:alpine --import-from=dir:"$ARTIFACTS_PATH"/redis-image --add-name=docker.io/library/redis:alpine --signature-policy="$INTEGRATION_ROOT"/policy.json
+# TODO: remove the code below for redis:alpine digested image id when
+#       https://github.com/kubernetes-incubator/cri-o/issues/531 is complete
+#       as the digested reference will be auto-stored when pulling the tag
+#       above
+	"$COPYIMG_BINARY" --root "$TESTDIR/crio" $STORAGE_OPTS --runroot "$TESTDIR/crio-run" --image-name=redis@sha256:03789f402b2ecfb98184bf128d180f398f81c63364948ff1454583b02442f73b --import-from=dir:"$ARTIFACTS_PATH"/redis-image-digest --add-name=docker.io/library/redis@sha256:03789f402b2ecfb98184bf128d180f398f81c63364948ff1454583b02442f73b --signature-policy="$INTEGRATION_ROOT"/policy.json
```
Comment (TomSweeneyRedHat) [inline test/helpers.bash]: Maybe nit, should it be COPYIMG or COPYING?  If COPYIMG, perhaps change it to COPY_IMG_BINARY?

**Comment by TomSweeneyRedHat** [comment]
LGTM, assuming happy tests.

File: test/helpers.bash (hunk 1 above)
Comment (runcom) [inline test/helpers.bash]: it's `COPYIMG`, let's folow up on rename though.

**Comment by mrunalp** [comment]
Looks good. Will wait for tests to complete.

**Comment by runcom** [comment]
all green except oom test @mrunalp - this PR should not impact kube tests though, feel free to merge.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**